explauto/interest_model/riac.py

```diff
@@ -0,0 +1,947 @@
+import time
+import numpy as np
+import matplotlib.pyplot as plt
+from heapq import heappop, heappush
+from scipy.spatial.kdtree import minkowski_distance_p
+
+from .interest_model import InterestModel
+from .competences import competence_exp, competence_dist
+from ..utils.utils import rand_bounds
+from ..utils.config import make_configuration
+
+
+
+class RiacInterest(InterestModel):
+    def __init__(self, 
+                 conf, 
+                 expl_dims, 
+                 max_points_per_region, 
+                 split_mode, 
+                 competence_measure, 
+                 progress_win_size, 
+                 progress_measure, 
+                 sampling_mode):
+        
+        self.conf = conf
+        self.bounds = self.conf.bounds[:, expl_dims]
+        self.competence_measure = competence_measure
+        
+        if progress_win_size >= max_points_per_region:
+            print "WARNING: progress_win_size should be < max_points_per_region"
+        
```

Comment:
Is this really only a Warning, or should it never happened? In this case, I will recommend instead something like:

```python
if progress_win_size >= max_points_per_region:
    raise ValueError("WARNING: progress_win_size should be < max_points_per_region")
```